### PR TITLE
ToggleSlider: enable selection for widget with data-apperance=slider

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Issue: N/A

Problem:
 ToggleSlider cannot be selected in Design Editor

Solution:
 Added selector for ToggleSlider component which allows
 select the widget.

Test case:
- build design editor
- build TAU
- copy TAU to sample app or make symlink (TAU/dist -> [APP]/libs/tau)
- add box
- add several toggle-slider to box
- try to select a slider

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
I have some problems with verification. Could you suggest verificaiton steps? In what kind of sample app I should try to put toggle slider. The samples which I try don't allow to put toggle slider

**Comment by TomaszLukawskiSam** [comment]
> I have some problems with verification. Could you suggest verificaiton steps? In what kind of sample app I should try to put toggle slider. The samples which I try don't allow to put toggle slider

This change depends to change in TAU (https://github.com/Samsung/TAU/pull/153)

File: tau-component-packages/components/toggleslider/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "label": "toggleslider",
-  "selector": ".ui-toggle-container, .ui-iot-toggleslider, .ui-iot-toggleswitch, .ui-toggleslider-base",
+  "selector": ".ui-toggle-container, .ui-iot-toggleslider, .ui-iot-toggleswitch, .ui-toggleslider-base, .ui-toggle-slider-container",
```
Comment (pkaczmarczy) [inline tau-component-packages/components/toggleslider/package.json]: We no longer use iot components. I think we can remove the iot related classes from the selector.

Comment (TomaszLukawskiSam) [reply tau-component-packages/components/toggleslider/package.json]: Done

**Pull request merged**